resources/data/nrf_connect_sdk-0.1.3.json

```diff
@@ -49,6 +49,62 @@
                 }
             ]
         },
+        {
+            "type": "Checkable",
+            "steps": [
+                {
+                    "type": "Step",
+                    "platforms": [
+                        { "platform": "darwin" }
+                    ],
+                    "title": "Installing additional tools for Connected Home over IP applications development",
+                    "description": [
+                        "_Note: If you are not interested in the [Connected Home over IP](https://www.connectedhomeip.com/) applications, you can ommit steps presented in this section._",
+                        "Download and extract [GN](https://gn.googlesource.com/gn/) build system executable archive used by the Connected Home over IP to generate output files:",
+                        {
+                            "type": "commands",
+                            "description": [
+                                "cd ~",
+                                "wget -O gn.zip https://chrome-infra-packages.appspot.com/dl/gn/gn/mac-amd64/+/latest",
+                                "unzip gn.zip gn",
+                                "rm gn.zip"
+                            ]
+                        },
+                        "Put GN location into the system PATH:",
+                        "```\necho \"export PATH=$PWD:$PATH\" >> $HOME/.bashrc\n```"
```

Comment:
I am not convinced this has the effect you intended. I don't think the value of $PWD will be the one you'd expect when `.bashrc` is evaluated.

Reply:
I'm not sure I understand. I would like to extract the archive and add path to the gn executable file to PATH, so assuming that user is currently in the directory where file was downloaded and extracted, $PWD will be just the desired location. In the .bashrc there will not be $PWD symbol, but just path passed through $PWD in the moment of calling echo. Why do you think it is not expected?

Reply:
In that case PATH will also be resolved when you call echo, and that is not what you would want to put in _.bashrc_.

Reply:
Ah yes, now I get what you mean. I took a look on commands in other examples and I tried to align to them. I also added `source ${HOME}/.bashrc` command to not force user reboot the system to update PATH. I noticed that in other places like here: https://github.com/NordicSemiconductor/pc-nrfconnect-gettingstarted/blob/master/resources/data/linux_zephyr_deps-0.9.5.json#L106 the PATH value is exported same way, as I previously did. Is there also a mistake?

Reply:
Yes, what you found there is also a mistake, probably copied over from the online docs.